Content.Server/Pinpointer/NavMapSystem.cs

```diff
@@ -87,6 +90,27 @@ private void OnNavMapSplit(ref GridSplitEvent args)
         RefreshGrid(comp, _gridQuery.GetComponent(args.Grid));
     }
 
+    private void OnNavMapBeaconMapInit(EntityUid uid, NavMapBeaconComponent component, MapInitEvent args)
```

Comment:
OnNavMapBeaconMapInit is defined twice, combine them into a single method.

Reply:
No longer relevant